datadog_cluster_agent/metadata.csv

```diff
@@ -1,66 +1,67 @@
-datadog.cluster_agent.external_metrics.datadog_metrics,gauge,,,,"The label valid is true if the DatadogMetric CR is valid, false otherwise",0,datadog_cluster_agent,external metrics datadog metrics,
-datadog.cluster_agent.external_metrics.delay_seconds,gauge,,second,,Freshness of the metric evaluated from querying Datadog,0,datadog_cluster_agent,external metrics delay,
-datadog.cluster_agent.external_metrics.processed_value,gauge,,,,Value processed from querying Datadog by metric,0,datadog_cluster_agent,external metrics processed,
-datadog.cluster_agent.go.goroutines,gauge,,,,Number of goroutines that currently exist,0,datadog_cluster_agent,go goroutines,
-datadog.cluster_agent.go.memstats.alloc_bytes,gauge,,byte,,Number of bytes allocated and still in use,0,datadog_cluster_agent,go memstats alloc bytes,
-datadog.cluster_agent.go.threads,gauge,,thread,,Number of OS threads created,0,datadog_cluster_agent,go threads,
-datadog.cluster_agent.kubernetes_apiserver.emitted_events,count,,,,"Datadog events emitted by the kubernetes_apiserver check",0,datadog_cluster_agent,datadog events events,
-datadog.cluster_agent.kubernetes_apiserver.kube_events,count,,,,"Kubernetes events processed by the kubernetes_apiserver check",0,datadog_cluster_agent,apiserver events,
-datadog.cluster_agent.language_detection_dca_handler.processed_requests,count,,,,"The number of process language detection requests processed by the handler",0,datadog_cluster_agent,language detection processed requests,
-datadog.cluster_agent.language_detection_patcher.patches,count,,,,"The number of patch requests sent by the patcher to the kube api server",0,datadog_cluster_agent,language detection patches,
-datadog.cluster_agent.secret_backend.elapsed,gauge,,millisecond,,The elapsed time of secret backend invocation,0,datadog_cluster_agent,secret backend elapsed time duration,
-datadog.cluster_agent.tagger.stored_entities,gauge,,,,Number of entities stored in the tagger,0,datadog_cluster_agent,tagger stored entities,
-datadog.cluster_agent.tagger.updated_entities,count,,,,Number of updates made to entities in the tagger,0,datadog_cluster_agent,tagger updated entities,
-datadog.cluster_agent.workloadmeta.events_received,count,,,,Number of events received by workloadmeta,0,datadog_cluster_agent,workloadmeta events received,
-datadog.cluster_agent.workloadmeta.notifications_sent,count,,,,Number of notifications sent by workloadmeta to its subscribers,0,datadog_cluster_agent,workloadmeta notifications sent,
-datadog.cluster_agent.workloadmeta.stored_entities,gauge,,,,Number of entities stored in workloadmeta,0,datadog_cluster_agent,workloadmeta stored entities,
-datadog.cluster_agent.workloadmeta.subscribers,gauge,,,,Number of workloadmeta subscribers,0,datadog_cluster_agent,workloadmeta subscribers,
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
```

Comment:
question: which metrics are using `sample_tags` currently?

Reply:
My last commit complained for me to run `ddev validate metadata datadog_cluster_agent --sync` and it produced this sample tags field (I have no idea why). I can go an manually remove it.

Reply:
Ah just found this PR https://github.com/DataDog/integrations-core/pull/17521; seems like`--sync` will automatically add the column, but for existing files it's fine with or without. Not sure if `sample_tags` should be added to all `metadata.csv` files going forward so can wait for agent-integrations to chime in before you make changes